src/java.base/share/classes/com/sun/crypto/provider/DHKeyAgreement.java

```diff
@@ -413,9 +405,8 @@ protected SecretKey engineGenerateSecret(String algorithm)
             int keysize = secret.length;
             if (keysize >= BlowfishConstants.BLOWFISH_MAX_KEYSIZE)
                 keysize = BlowfishConstants.BLOWFISH_MAX_KEYSIZE;
-            SecretKeySpec skey = new SecretKeySpec(secret, 0, keysize,
+            return new SecretKeySpec(secret, 0, keysize,
```

Comment:
nit: merge the following line, i.e. "Blowfish", with this line?

Reply:
Done.